rocAL/rocAL/include/text_file_meta_data_reader.h

```diff
@@ -33,6 +33,7 @@ class TextFileMetaDataReader: public MetaDataReader
     void read_all(const std::string& path) override;
     void release(std::string image_name);
     void release() override;
+    void set_timestamps_bool() override { return; }
```

Comment:
why we need this function here?

Reply:
set_timestamps_bool will be called in [master_graph.cpp](https://github.com/fiona-gladwin/MIVisionX/blob/695f3083ddc8f7575853be502786f971a1b23803/rocAL/rocAL/source/master_graph.cpp#L1117) and it is introduced a virtual function in [MetaDataReader ](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/695f3083ddc8f7575853be502786f971a1b23803/rocAL/rocAL/include/meta_data_reader.h#L83)hence we have defined it in all the meta data readers derived from the class class.